app/apis/video.py

```diff
@@ -0,0 +1,95 @@
+from flask import request, Response
+import json
+from flask_restplus import Api, Resource, Namespace
+from app.database.models.author import AuthorModel
+from app.database.models.videos import VideoModel
+from app.database.models.section import SectionModel
+from app.database.models.category import CategoryModel
+from app.database.sqlalchemy_extension import db
+
+api = Namespace('video', description='Video Library')
+
+@api.route('/latest')
+class VideoLibrary(Resource):
+    def get(self):
+        with open('./content/latests.json') as f:
+            return json.loads(f.read())
+        return jsonify({'message': 'File does not exist.'})
+
+@api.route('/')
+class VideoAuthor(Resource):
+    def get(self):
+        
+        ''' To demonstrate the bidirectional and many-to-many relations of database. 
+        Visit localhost:5000/video 
+        Below is the sample code to debug the proper functionality of backref and relationships.  '''
+        author1 = AuthorModel(name="Jill")
```

Comment:
What are those hardcoded values?

Reply:
This is for testing purposes only. If you open that route and look into the database. You can see all the tables with data. There is no real significance of these data. 
If you are are satisfied with the implementation then I'll remove these